extensions/openshift-client/deployment/src/main/java/io/quarkus/openshift/client/deployment/OpenShiftClientProcessor.java

```diff
@@ -0,0 +1,137 @@
+package io.quarkus.openshift.client.deployment;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.inject.Inject;
+
+import org.jboss.jandex.DotName;
+import org.jboss.jandex.Type;
+import org.jboss.logging.Logger;
+
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.deployment.Feature;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.ApplicationIndexBuildItem;
+import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
+import io.quarkus.deployment.builditem.ExtensionSslNativeSupportBuildItem;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.ReflectiveClassBuildItem;
+import io.quarkus.deployment.util.JandexUtil;
+import io.quarkus.jackson.deployment.IgnoreJsonDeserializeClassBuildItem;
+import io.quarkus.kubernetes.client.deployment.KubernetesClientProcessor;
+import io.quarkus.kubernetes.spi.KubernetesRoleBuildItem;
+import io.quarkus.openshift.client.runtime.OpenShiftClientProducer;
+
+public class OpenShiftClientProcessor {
```

Comment:
This seems to have been copied from `KubernetesClientProcessor`. Since this extension depends on kubernetes-client, this is not needed. It should contain only what's necessary for `OpenShiftClient`